.github/workflows/test.yml

```diff
@@ -61,8 +64,8 @@ jobs:
           coverage.xml
           build/coverage/html/
     - name: Publish package on PyPI
-      if: matrix.python-version == 3.9 && github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@v1.4.2
```

Comment:
I don't know this particular GitHub Action enough to know if this is a good change or not, but it does seem unrelated to the commit message.

Reply:
It looks like the new form here (`@release/v1`) is the current recommendation from upstream so I think it is a good change.  But probably it should be a deliberate change.